packages/diagrams/frontend/sirius-components-diagrams/src/sprotty/views/LabelView.tsx

```diff
@@ -88,7 +89,11 @@ export class LabelView extends SLabelView {
 
     const vnode = (
       <g attrs-data-testid={`Label - ${label.text}`}>
-        {iconURL ? <image href={iconURL} y={iconVerticalOffset} x="-20" style={styleIcon} /> : ''}
+        {iconURL.length > 0
+          ? iconURL.map((icon, index) => (
```

Comment:
Trying to create an Entity on a (Sprotty-base) _Domain_ diagram I get:
```
Uncaught TypeError: iconURL.map is not a function
    at LabelView.render (sirius-components-diagrams.es.js:7373:123)
```
and nothing appears on my diagram.

Reply:
The root cause is in `convertDiagram.tsx`, see my comment there.

Reply:
Fixed by https://github.com/eclipse-sirius/sirius-web/pull/2400/commits/4d2b0daf6a6110fa3ba1f26adf24b3fa437b6527 (to be squashed).